### DISCUSSION TITLE
v2.6.2

### DISCUSSION THREAD

**Discussion by mashb1t** [mashb1t/Fooocus#66] [Announcements]
## What's Changed
* fix: add positive prompt if styles don't have a prompt placeholder by @mashb1t in https://github.com/lllyasviel/Fooocus/pull/3372
* docs: update numbering of basic debug procedure in issue template by @mashb1t in https://github.com/lllyasviel/Fooocus/pull/3376
* feat: extend config settings for input image by @mashb1t in https://github.com/lllyasviel/Fooocus/pull/3382
* feat: count image count index from 1 by @mashb1t in https://github.com/lllyasviel/Fooocus/pull/3383

**Full Changelog**: https://github.com/mashb1t/Fooocus/compare/v2.6.1...v2.6.2

<hr /><em>This discussion was created from the release <a href='https://github.com/mashb1t/Fooocus/releases/tag/v2.6.2'>v2.6.2</a>.</em>

**Comment by soyv4** [comment]
Welcome mashb1t! I replaced the control-lora-canny-rank 128 model with diffusion_pytorch_model.safetensors, something had to be left out, protested the model works! https://huggingface.co/xinsir/controlnet-union-sdxl-1.0

**Reply by mashb1t** [reply]
@soyv4 can you provide a PR to debug this together? Thanks!

**Reply by soyv4** [reply]
> @soyv4можете ли вы предоставить PR для совместной отладки? Спасибо!

I am not an expert in this field, I threw the downloaded model and renamed it to control-lora-canny-rank 128, and it works, please check

**Reply by cubiwan** [reply]
I have tested using openpose images and it works, no code changes are necessary:
* Download diffusion_pytorch_model.safetensors
* Save in controlnet folder
* Rename to control-lora-canny-rank128.safetensors
* Launch Fooocus
* Advanced tab, Control tab, check Skip Preprocessors
* Image Prompt tab with an openpose skeleton image (512x512)
* Select PyraCanny option
* Select Aspect Ratio 1:1 (with another the generated image appears cut off)
* Write a prompt and generate image.
